charms/istio-pilot/charmcraft.yaml

```diff
@@ -9,6 +9,7 @@ bases:
 parts:
   charm:
     charm-python-packages: [setuptools, pip]
+    build-packages: [git]
```

Comment:
I don't know if I've had to use this before - what does this add?

Reply:
We need it to be able to fetch [this](https://github.com/ca-scribner/lightkube/tree/add-implicit-model-generation) experimental LK version